README.md

```diff
@@ -16,7 +16,7 @@ Add issues or explore the project on [github](https://github.com/netresearch/t3x
     1. with composer from [packagist](https://packagist.org/packages/netresearch/rte-ckeditor-image)
     
         ```shell
-        composer require netresearch/rte-ckeditor-image
+        composer req netresearch/rte-ckeditor-image:dev-TYPO3_10.x
```

Comment:
should be skipped in merge if possible - even dunno how to do this.

Reply:
Right, gonna change it back.